packages/npm2gpr/examples/example-1.js

```diff
@@ -0,0 +1,8 @@
+const npm2gpr = require("../");
+
+async function main(){
```

Comment:
This is an example used to teach users how to use this module. Please add detailed comments explaining what's going on.